README.md

```diff
@@ -16,6 +14,21 @@ Custom Resource Definitions(CRDs) and Kubernetes-native tooling.
 
 [**Features**](#features) | [**Get started**](#get-started) | [**Documentation**](#documentation) | [**main branch builds**](#main-branch-builds) | [**Seeking help**](#seeking-help)
 
+## ✨ KIC 2.0 is in public beta! ✨
+
+We're happy to announce that Kong Ingress Controller 2.0 is in public beta.
+
+This new release comes with a major internal rearchitecture, and the following added features:
+- Support for UDP ingress resources and the new `UDPIngress` CRD,
+- Ingress controller runtime metrics compatible with Prometheus, for easier monitoring and alerting.
+
+For a quick start, just deploy KIC 2.0 beta to your cluster:
+```console
```

Comment:
```suggestion
```deployment
```

Reply:
Why?

Per GH documentation, there is no such syntax coloring as `deployment`.
https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks
https://github.com/github/linguist/blob/master/lib/linguist/languages.yml

Reply:
Deployment is more accurate, user could execute the tool either on a console or on a terminal.
This is a "README.md", no a code block.